templates/kapeta/block-type-frontend/browser-routes.tsx.hbs

```diff
@@ -0,0 +1,38 @@
+//#FILENAME:src/browser/routes.tsx:create-only
```

Comment:
Make it "write-never" if it's only intended for the AI

Reply:
I feel like there's a concept missing - right now we combine write-never and `ai-type` into special handling.
How about a new file mode that works similar to `{{#ai-context}}` ? `:ai-only`?

Reply:
But we probably dont want to generate this file in "normal" mode? That's why we use "write-never"

Reply:
But yes we could improve our "ai-type" so that it's not written out in the template